### PR TITLE
Add pyega3

### PR DESCRIPTION
**Pull request by AlexanderVi**
* [ *] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ *] This PR adds a new recipe.
* [ *] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/pyega3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - pyega3 = pyega3.pyega3:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
```
Comment (bgruening) [inline recipes/pyega3/meta.yaml]: only once :)

File: recipes/pyega3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+    - python
+    - requests
```
Comment (bgruening) [inline recipes/pyega3/meta.yaml]: I don't think requests and tqdm are needed in the host env, isn't it?

**Comment by AlexanderVi** [comment]
Thanks for your comments,Björn! I've fixed the host section. Also I had to remove 'noarch' (otherwise tests fail).
PS. Do you know ,by chance, is there any way to specify not exact version number, but some kind of wildcard saying "take the latest available on PyPi repo" ?

**Comment by johanneskoester** [comment]
The explicit version is necessary, because it might be that dependencies change in future versions. We do not regularly rebuild the packages, but rather rely on PRs for version updates. However, we actually plan an update bot that tracks pypi, so that recipes are updated automatically for new versions.

**Pull request merged**